sal_interfaces/TunableLaser/TunableLaser_Commands.xml

```diff
@@ -33,9 +33,9 @@
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>stopPropagateLaser</EFDB_Name>
```

Comment:
called value, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused."